timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -291,7 +298,7 @@ export default {
     return {
       showSketchMetadata: false,
       navigationDrawer: {
-        width: 400,
+        width: 370,
```

Comment:
Changing the width of the left panel back to 370 will result in an odd line break in the left panel when the pagination is used. (The next page arrow gets moved to a new line, see screenshot below)
![page](https://user-images.githubusercontent.com/99879757/222695315-0231837b-db4c-4f18-90dd-9aedeac3d7a3.png)

Reply:
Ah, makes sense. Change it to 400 as default